.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,105 @@
+name: Build
+
+on:
+    push:
+        branches:
+            - master
+        tags:
+            - "v*"
+    pull_request:
+        branches:
+            - master
+
+jobs:
+    Build:
+        strategy:
+            matrix:
+                options:
+                    - os: ubuntu-latest
```

Comment:
shouldn't that be better a specific version of ubuntu? Same with windows below?

Reply:
latest is fine 
https://help.github.com/en/actions/automating-your-workflow-with-github-actions/virtual-environments-for-github-hosted-runners